test/setup.ts

```diff
@@ -48,6 +45,14 @@ export const setupRecording = (): Context =>
             "auth api/login caches user data",
             "auth api/refreshes the auth token",
             "auth api/logout clears user cache",
+            "auth api/verifies if token is valid",
+            "user api/sends request to change user's email",
+            "user api/updates the user's first name",
+            "user api/creates the user's account address",
+            "user api/updates the user's account address",
+            "user api/sets address as a default billing address",
+            "user api/deletes user address",
+            "user api/deletes user account",
```

Comment:
Such lists doesnt scale well and are area for issues. @jwm0 Ideas how we can refactor it? Remove fields like email and redirect URL from all test matches?

Reply:
I agree, let's just simply maintain a blacklist of all variables that need to be deleted before creating the recording and use it everywhere:
```
const variablesBlacklist = [
  "email",
  "password",
  "redirectUrl",
  "newPassword",
  "oldPassword",
  "newEmail"
];

body(body, req): string {
  const json = JSON.parse(body);

  variablesBlacklist.forEach((key) => {
    delete obj.variables[key];
  })

  return JSON.stringify(json);
}
```